src/include/distributed/citus_nodefuncs.h

```diff
@@ -47,19 +35,10 @@ extern void RegisterNodes(void);
  * different from before.
  */
 
-#if (PG_VERSION_NUM >= 90600)
 #define READFUNC_ARGS struct ExtensibleNode *node
 #define READFUNC_RET void
-#else
-#define READFUNC_ARGS void
-#define READFUNC_RET Node *
-#endif
 
-#if (PG_VERSION_NUM >= 90600)
 #define OUTFUNC_ARGS StringInfo str, const struct ExtensibleNode *raw_node
-#else
-#define OUTFUNC_ARGS StringInfo str, const Node *raw_node
-#endif
 
```

Comment:
> Maybe a separate commit?
Can, but not enthusiastic.

> It's a find-and-replace, basically, right?

Yea, pretty much.It makes the lines a good bit longer tho.